qmcpy/discrete_distribution/lattice/lattice.py

```diff
@@ -40,6 +50,24 @@ class Lattice(LD):
            [0.5 , 0.5 ],
            [0.25, 0.75],
            [0.75, 0.25]])
+    >>> Lattice(dimension=2,randomize=False,seed=182,generating_vector = 23).gen_samples(8, warn=False)
+    array([[0.   , 0.   ],
+           [0.5  , 0.5  ],
+           [0.25 , 0.25 ],
+           [0.75 , 0.75 ],
+           [0.125, 0.125],
+           [0.625, 0.625],
+           [0.375, 0.375],
+           [0.875, 0.875]])
```

Comment:
doctests are more for illustrating use than for testing capabilities. This code snippet and the following one do pretty much the same thing, so maybe keep the 4 dimensional one and remove the 2d example.